tests/PHPUnit/Integration/EmailValidatorTest.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Tests\Integration;
+
+use Piwik\Piwik;
+use Piwik\Tests\Framework\TestCase\IntegrationTestCase;
+
+/**
+ * @group Core
+ */
+class EmailValidatorTest extends IntegrationTestCase
```

Comment:
Is there a reason this is an integration test? Note: currently the IntegrationTestCase base class will create a database before tests start and reset it before each test, which slows things down.